packages/bsky/tests/admin/moderation.test.ts

```diff
@@ -960,6 +962,86 @@ describe('moderation', () => {
       )
     })
 
+    it('fans out repo takedowns to pds', async () => {
+      const { data: action } =
+        await agent.api.com.atproto.admin.takeModerationAction(
+          {
+            action: TAKEDOWN,
+            createdBy: 'did:example:moderator',
+            reason: 'Y',
+            subject: {
+              $type: 'com.atproto.admin.defs#repoRef',
+              did: sc.dids.bob,
+            },
+          },
+          {
+            encoding: 'application/json',
+            headers: network.bsky.adminAuthHeaders(),
+          },
+        )
+
+      const res1 = await pdsAgent.api.com.atproto.admin.getSubjectStatus(
+        {
+          did: sc.dids.bob,
+        },
+        { headers: network.pds.adminAuthHeaders() },
+      )
+      expect(res1.data.takedown?.applied).toBe(true)
+
+      // cleanup
+      await reverse(action.id)
+
+      const res2 = await pdsAgent.api.com.atproto.admin.getSubjectStatus(
+        {
+          did: sc.dids.bob,
+        },
+        { headers: network.pds.adminAuthHeaders() },
+      )
+      expect(res2.data.takedown?.applied).toBe(false)
+    })
+
+    it('fans out record takedowns to pds', async () => {
+      const post = sc.posts[sc.dids.bob][0]
+      const uri = post.ref.uriStr
+      const cid = post.ref.cidStr
+      const { data: action } =
+        await agent.api.com.atproto.admin.takeModerationAction(
+          {
+            action: TAKEDOWN,
+            createdBy: 'did:example:moderator',
+            reason: 'Y',
+            subject: {
+              $type: 'com.atproto.repo.strongRef',
+              uri,
+              cid,
+            },
+          },
+          {
+            encoding: 'application/json',
+            headers: network.bsky.adminAuthHeaders(),
+          },
+        )
+
+      const res1 = await pdsAgent.api.com.atproto.admin.getSubjectStatus(
+        {
+          uri,
+        },
+        { headers: network.pds.adminAuthHeaders() },
+      )
```

Comment:
Truly no problem here, but I bet you could get away with this if you like 😏 
```suggestion
      const res1 = await pdsAgent.api.com.atproto.admin.getSubjectStatus(
        { uri },
        { headers: network.pds.adminAuthHeaders() },
      )
```

Reply:
lol yup good looks - leftover from refactoring 😛